src/metrics/registry_disk.go

```diff
@@ -22,38 +22,38 @@ func (this RegistryMetric) GetEventHandler() func(interface{}) error {
 			return err
 		}
 		this.HandleData(
-			registryData,
+			registryDiskData,
```

Comment:
Again, not related to the CLOUD-799, but needed to be renamed to make things less confusing.